QUEST.tsv

```diff
@@ -773,101 +773,101 @@ QUEST_20150317_000772	There are some old pots among the new pots. {nl}Which one
 QUEST_20150317_000779	Found a very old clay tablet.
 QUEST_20150317_000780	There is nothing. 
-QUEST_20150317_000781	Seems like you might break the pot if you handle it wrong.
+QUEST_20150317_000781	Seems like you might break the pot if you mishandle it.
 QUEST_20150317_000782	Empty.
 QUEST_20150317_000783	A bug came out. 
 QUEST_20150317_000784	A half eaten potato came out. 
 QUEST_20150317_000785	A clay tablet came out, but it's too worn out to be read.
 QUEST_20150317_000786	Importance of Rest
-QUEST_20150317_000787	Young men nowadays are so restless. {nl}They try to do everything while standing up. {nl}
-QUEST_20150317_000788	Whether it be making weapons, making potions, or even cooking, they try to do it while standing. {nl}You must calm yourself and sit in a comfortable position. {nl}
-QUEST_20150317_000789	Sit down and prepare all the tools and materials you need. {nl}Remember that making comes after preparations. 
+QUEST_20150317_000787	Young men nowadays are so restless. {nl}They try to do everything while standing up.
+QUEST_20150317_000788	Whether it be making weapons, making potions, or even cooking, they try to do it while standing. {nl}You must calm yourself and sit in a comfortable position.
+QUEST_20150317_000789	Sit down and prepare all the tools and materials you need. {nl}Remember that creation comes after preparations. 
 QUEST_20150317_000790	Goddess Austeja
-QUEST_20150317_000791	Thank you again in the name of the Goddess. {nl}Now I will follow the stars in search of Goddess Ausrine. {nl}
-QUEST_20150317_000792	It may be better to guard the seal. {nl}But I believe finding all the lost sisters is the solution to all this. {nl}
+QUEST_20150317_000791	Thank you again in the name of the Goddess. {nl}Now I will follow the stars in search of Goddess Ausrine.
+QUEST_20150317_000792	It may be better to guard the seal. {nl}But I believe finding all the lost sisters is the solution to all this.
 QUEST_20150317_000793	I wish you well… {nl}And may you and this land be blessed.
 QUEST_20150317_000794	Right. I think so too. {nl}The monsters are definitely preying on our honey. {nl}But it is better to have evidence on hand, so bring a piece of beehive {nl}from Saltus Bee Farm.
 QUEST_20150317_000795	Coward
-QUEST_20150317_000796	They call me a coward, but I can't do anything. {nl}Because I'm really scared.
-QUEST_20150317_000797	You're the guy the bees were whispering about. {nl}I am the Goddess of Fate, Austeja. 
-QUEST_20150317_000798	The seal of this farmland has stabilized, thanks to you. {nl}But the other Revelators have lost themselves due to the evil force flowing from the Spring Light Woods.{nl}
-QUEST_20150317_000799	I am using all my powers to block the seal from opening any further… {nl}So I can't move away from here, let along save them. {nl}
-QUEST_20150317_000800	Therefore… Please save them and the Spring Light Woods.
+QUEST_20150317_000796	I'm called a coward. I can't do anything. I'm really scared.
```

Comment:
I would still leave in "but" in order to stress the contradiction that this person is making, to being called a coward.